docs-ref-services/latest/msal-python-overview.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Microsoft Authentication Library (MSAL) for Python overview
```

Comment:
Model this after https://docs.microsoft.com/python/api/overview/azure/cosmos-readme?view=azure-python

You might need to get some input from Ray, too, but ping me if you need help.

Reply:
Here's another set of guidance you can use:

https://review.docs.microsoft.com/en-us/help/contribute/contribute-how-to-write-library-quickstart?branch=master